cmd/common.go

```diff
@@ -46,3 +63,20 @@ func addCommonFlagsForGit(cmd *cobra.Command, defaults *cfg.Configuration) {
 	cmd.PersistentFlags().String("sort", defaults.Git.SortCriteria,
 		fmt.Sprintf("Sort git tags by criteria. Only effective with --tags. Allowed values: [%s, %s]", git.SortOptionVersion, git.SortOptionAlphabetic))
 }
+
+func listImages() error {
+	ns, err := kubernetes.Namespace()
+	if err != nil {
+		return err
+	}
+	imageStreams, err := openshift.ListImageStreams(ns)
+	if err != nil {
+		return err
+	}
+	imageNames := []string{}
+	for _, image := range imageStreams {
+		imageNames = append(imageNames, image.Name)
+	}
+	log.WithField("project", ns).WithField("images", imageNames).Info("Please select an image. The following images are available")
```

Comment:
With 2 fields this is borderline acceptable :) I think `log.WithFields(log.Fields{...}` is a little more readable with more than 1 field when split over multiple lines.